### ISSUE TITLE
'Namespace' object has no attribute 'set_cfgs'

### ISSUE THREAD

**Issue by drickzsy** [chaoqichen/HTCN#22]
''Loading pretrained weights from /home/zsy/HTCN-master/data/pretrained_model/resnet101_caffe.pth
Segmentation fault''
Please help me!

**Comment by wangm-ting** [comment]
if args.set_cfgs is not None:
AttributeError: 'Namespace' object has no attribute 'set_cfgs'
I also have this problem. Have you solved it?

**Comment by drickzsy** [comment]
> if args.set_cfgs is not None:
> AttributeError: 'Namespace' object has no attribute 'set_cfgs'
> I also have this problem. Have you solved it?

Yes, it's not really hard. This error maybe caused by your wrong cfgs or mismatched cfgs. You can carefully find it.

**Comment by wangm-ting** [comment]
what is your version of python and cfgs? I tried , but doesnot work.

**Comment by chengxy1998** [comment]
这可能是你运行代码时所配置的参数与parser_func.py中的参数设定不一致引起的，比如你的数据集名称没有按照parser_func.py中所设定的那样。
